bindings/python/examples/how_tos/accounts_and_addresses/consolidate_outputs.py

```diff
@@ -14,59 +14,61 @@
 
-account = wallet.get_account('Alice')
-
-# Sync account to make sure account is updated with outputs from previous
+# Sync wallet to make sure account is updated with outputs from previous
 # examples.
-account.sync()
+wallet.sync()
 print('Account synced')
 
 # List unspent outputs before consolidation.
 # The output we created with example `request_funds` and the basic output from `mint` have only one
 # unlock condition and it is an `AddressUnlockCondition`, and so they are valid for consolidation. They have the
 # same `AddressUnlockCondition`(the address of the wallet), so they will be consolidated into one
 # output.
-outputs = account.unspent_outputs()
+outputs = wallet.unspent_outputs()
 print('Outputs BEFORE consolidation:')
 
 for i, output_data in enumerate(outputs):
     print(f'OUTPUT #{i}')
     print(
         f'- address: #{Utils.hex_to_bech32(output_data.address.pub_key_hash, "rms")}')
     print(f'- amount: #{output_data.output.amount}')
-    print(f'- native tokens: #{}', [feature for feature in output_data.output.features if feature.type
-                                    == FeatureType.NativeToken])
+
+    native_tokens = [feature for feature in output_data.output.features if feature.type == FeatureType.NativeToken]
+    first_native_token = next(iter(native_tokens), None)
+    print(f'- native tokens: #{first_native_token}')
 
 print('Sending consolidation transaction...')
 
 # Consolidate unspent outputs and print the consolidation transaction ID
 # Set `force` to true to force the consolidation even though the
 # `output_threshold` isn't reached.
-transaction = account.consolidate_outputs(ConsolidationParams(force=True))
+transaction = wallet.consolidate_outputs(ConsolidationParams(force=True))
 print('Transaction sent: ', transaction.transaction_id)
 
 # Wait for the consolidation transaction to get confirmed
-block_id = account.reissue_transaction_until_included(
+block_id = wallet.reissue_transaction_until_included(
     transaction.transaction_id)
 
 print(
     f'Transaction included: {os.environ["EXPLORER_URL"]}/block/{block_id}'
 )
 
 # Sync account
-account.sync()
+wallet.sync()
 print('Account synced')
 
 # Outputs after consolidation
-outputs = account.unspent_outputs()
+outputs = wallet.unspent_outputs()
 print('Outputs AFTER consolidation:')
 for i, output_data in enumerate(outputs):
     print(f'OUTPUT #{i}')
     print(
         f'- address: #{Utils.hex_to_bech32(output_data.address.pub_key_hash, "rms")}')
     print(f'- amount: #{output_data.output.amount}')
-    print(f'- native tokens: #{}', [feature for feature in output_data.output.features if feature.type
-                                    == FeatureType.NativeToken])
+
+    native_tokens = [feature for feature in output_data.output.features if feature.type == FeatureType.NativeToken]
+    first_native_token = next(iter(native_tokens), None)
+    print(f'- native tokens: #{first_native_token}')
```

Comment:
why just the first? :thinking: If so, then consider changing the printed string to `- first native token`.